Bottom: 90d25b8276c78150b68fd32e082334d1a97d451b
Top:    cc93e0fa125f9f1892306d94aa785e9a125b8a35
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-16 16:16:59 +0800

Refresh of macvtap-xdp-fast-rx-support

---

diff --git a/drivers/net/macvtap.c b/drivers/net/macvtap.c
index 280c9a0..d7f3c55 100644
--- a/drivers/net/macvtap.c
+++ b/drivers/net/macvtap.c
@@ -353,6 +353,13 @@ static void macvtap_del_queues(struct net_device *dev)
 	vlan->numvtaps = MAX_MACVTAP_QUEUES;
 }
 
+static macvtap_xdp_rx(const struct sk_buff *skb, const struct bfp_insn *filter)
+{
+	struct xdp_buff *buff = (struct xdp_buff *)skb;
+
+	if (__ptr_ring_full(&
+}
+
 static rx_handler_result_t macvtap_handle_frame(struct sk_buff **pskb)
 {
 	struct sk_buff *skb = *pskb;
